src/containers/CustomResourceDefinition/CustomResourceDefinition.js

```diff
@@ -72,12 +74,21 @@ export /* istanbul ignore next */ class CustomResourceDefinition extends Compone
       case 'clustertasks':
         return this.props.fetchClusterTask(name);
       default:
-        return Promise.resolve();
+        return getCustomResource({
+          group,
+          version,
+          type,
+          namespace,
+          name
+        }).then(resource => {
+          this.setState({ loading: false, resource });
```

Comment:
no need to set `loading: false` here as it's already done in `componentDidMount/Update`